blocks/components/editable/index.js

```diff
@@ -127,6 +134,19 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		const formatMap = {
+			strong: 'bold',
+			em: 'italic'
+		};
```

Comment:
I wonder if we should do the mapping before calling the `onFormatChange` instead